efel/pyfeatures/pyfeatures.py

```diff
@@ -28,39 +28,62 @@
 
 import numpy
 import efel.cppcore
+from scipy.ndimage.filters import gaussian_filter1d
+from numpy.fft import *
 
 
 all_pyfeatures = [
     'voltage',
     'time',
-    'current',
     'ISIs',
     'initburst_sahp',
     'initburst_sahp_vb',
     'initburst_sahp_ssse',
-    'depol_block']
+    'depol_block',
+    'current',
+    'impedance']
 
 
 def voltage():
     """Get voltage trace"""
     return _get_cpp_feature("voltage")
 
 
+def current():
+    """Get voltage trace"""
+    return _get_cpp_feature("current")
+
+
 def time():
     """Get time trace"""
     return _get_cpp_feature("time")
 
 
-def current():
-    """Get current trace"""
-    return _get_cpp_feature("current")
+def impedance():
+    volta = _get_cpp_feature("voltage")
```

Comment:
use longer variable names (voltage), one can use the voltage() function defined above?